code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -688,3 +688,14 @@
 	construction_time=150
 	build_path = /obj/item/drone_shell
 	category = list("Misc")
+
+/datum/design/toolarm
```

Comment:
You should put this in https://github.com/tgstation/-tg-station/blob/master/code/modules/research/designs/medical_designs.dm with the others, and make it usable for both mech fab and protolathe. Any of the implants made on the protolathe are usually brought to robotics for implantation anyway.